src/zenml/zen_stores/rest_zen_store.py

```diff
@@ -2028,10 +2034,5 @@ def _delete_resource(
         self.delete(f"{route}/{str(resource_id)}")
 
     def _sync_runs(self) -> None:
-        """Syncs runs from MLMD.
-
-        Raises:
-            NotImplementedError: This internal method may not be called on a
-                `RestZenStore`.
-        """
-        raise NotImplementedError
+        """Syncs runs from MLMD."""
+        self.get(METADATA_SYNC)
```

Comment:
I think we should have some error handling on this, if it fails, we wouldn't want to throw an error for the user, the user will be helpless anyways, no need to bother them?

Reply:
Good idea, I added error handling inside the endpoint so we only log an exception to the server logs instead of raising it